extensions/cornerstone/src/CornerstoneViewportDownloadForm.js

```diff
@@ -115,6 +115,12 @@ const CornerstoneViewportDownloadForm = ({ onClose, activeViewportIndex }) => {
       const URLObj = window.URL || window.webkitURL;
       const a = document.createElement('a');
       a.href = URLObj.createObjectURL(blob);
+      window.downloadUrl = URLObj.createObjectURL(blob);
+      // Do not open the download dialog if Cypress test is being executed
+      if (window.Cypress) {
+        console.log(window.downloadUrl);
+        return;
+      }
```

Comment:
I'm hesitant to bake test specific code like this in.
If this is the only way to test this functionality, I'm okay with us foregoing it, or doing a simpler integration test.

Reply:
Yes, that's the only way of testing the `Download` button. We need this not only to expose the url but also to block the dialog event which Cypress cannot handle.
Without this, we don't have any information (url, requests, ...) to verify that the `Download` button is working.

Reply:
Unfortunately, I think that means we'll need to not test this specific functionality with Cypress.
We'll have to explore integration/unit tests, and assume the browser does it's job if the button is clicked here.